src/main/java/org/geysermc/discordbot/commands/moderation/BanCommand.java

```diff
@@ -145,8 +146,8 @@ protected void execute(CommandEvent event) {
             }, throwable -> {});
         }
 
-        // Ban user
-        event.getGuild().ban(user, delDays, String.join(" ", args)).queue();
+        // Ban user, updated in jda 5.0.0-alpha.20.
```

Comment:
I think this is an unnecessary comment - we can just check the Git diff if need be.

Reply:
Removed! :)